strr-host-pm-web/app/pages/dashboard/[applicationId].vue

```diff
@@ -32,6 +34,33 @@ onMounted(async () => {
     application.value?.header,
     ApplicationType.HOST
   )
+
+  // TODO: Remove after QA, registration number 96950160599768
+  const isTestRenewalApp = process.env.NODE_ENV === 'development' && registration.value?.id === 308
```

Comment:
@shaangill025 let me know what you think of adding a specific Reg in Dev for testing. Or wait until API is ready?